projects/angular-editor/src/lib/angular-editor-toolbar.component.html

```diff
@@ -121,7 +121,7 @@
   </div>
   <div class="angular-editor-toolbar-set">
     <button [id]="'link-'+id" type="button" class="angular-editor-button" (click)="insertUrl()"
-            title="Insert Link" [disabled]="htmlMode" tabindex="-1">
+            title="Insert Link" [disabled]="isLinkButtonDisabled" tabindex="-1">
```

Comment:
Hi! Thanks for PR, but this changes not working for me. Button still disabled when text was selected.

Reply:
Huh, will check a bit later, but when I tested it - it worked great.
When I will done with check - I will reply again.

Reply:
@kolkov 
Did you click outside editor after selection? It's important because it's relates to https://github.com/kolkov/angular-editor/issues/85

Reply:
@kolkov 
Do you have telegram or skype? I guess it would better if we move to any of them to communicate faster.

Reply:
I think we need to use mouseUp event to check if text is selected.
https://stackoverflow.com/questions/3545018/selected-text-event-trigger-in-javascript
Yes, I have Skype and Telegram.

Reply:
Or <textarea onselect="message()"

Reply:
@kolkov 
Already made changes are commited. Check it.
My telegram is @srjkaa

Reply:
Thanks! This worked, but if I move mouse outside the text area when select text, this not working.

Reply:
@kolkov 
Please, provide your telegram nickname for move our conversation in telegram.

Reply:
@kolkov 
Uploaded video with PoC in Yandex.disk: https://yadi.sk/i/YTLf8-uVrrZ6Ew
Just download and check it.
And if you can please provide me a case when it works not as expected.

Reply:
Ohh my bad, My local branche was not updated with latest changs. Now all good.

Reply:
But now, if I select text and move mouse pointer outside text area and click, selection lost, but button still enabled until I click back to textarea. It's not good, as I think...

Reply:
@kolkov It relates to: https://github.com/kolkov/angular-editor/issues/85
With fix issue above it will be ok.

Reply:
https://t.me/ngSoviet/139757 about this problem